numpy/doc/byteswapping.py

```diff
@@ -31,16 +31,16 @@
 3 + 2, the 4 bytes in memory would contain respectively: 0, 1, 3, 2.
 The bytes I have loaded from the file would have these contents:
 
->>> big_end_str = chr(0) + chr(1) + chr(3) + chr(2)
->>> big_end_str
-'\\x00\\x01\\x03\\x02'
+>>> big_end_array = bytearray([0,1,3,2])
+>>> big_end_array
+bytearray(b'\\x00\\x01\\x03\\x02')
```

Comment:
Calling this `_array` is definitely better than `_str` was, but it's a little similar to the `_arr` used below. Can you use `_buffer` or `_bytes` instead?